CinemaCat.Api/Program.cs

```diff
@@ -11,12 +10,10 @@
 builder.Services.AddSwaggerGen();
 builder.Services.AddControllers();
 
-builder.Services.AddDataBase(builder.Configuration);
-builder.Services.AddAzureClients(clientBuilder =>
-{
-    clientBuilder.AddBlobServiceClient(builder.Configuration.GetConnectionString("BlobStorage"), preferMsi: true);
-});
-builder.Services.AddMediatR(cfg => cfg.RegisterServicesFromAssembly(Assembly.GetExecutingAssembly()));
+builder.Services.Configure<MongoConfiguration>(builder.Configuration.GetSection(MongoConfiguration.SectionName));
```

Comment:
now that we moved the MongoConfiguration class into the Infrastructure project it becomes less logic to Configure it here.
The best option as I see it to move all configuraiton to the App layer. But we also can have the MongoConfiguration setup up entirely in the Infrastructure layer as it's used only there